.gitignore

```diff
@@ -13,12 +13,13 @@
 /tutorials/**/index_files/*
 Testing/
 /*/*/jl_*/
-/Manifest.toml
-/test/Manifest.toml
 .vscode
 _freeze
 _site
 .quarto
 /.quarto/
 changelog.qmd
-versions.qmd
\ No newline at end of file
+versions.qmd
+tmp.gif
+.venv
+venv
```

Comment:
It doesn't need to be there :) I added this a while back and only realised why I needed it fairly recently. Quarto (since v1.5) has a native Julia runner, but pre-1.5 it uses Jupyter and so requires a venv.

I guess technically it's useful for anybody who clones the repo and attempts to build it with Quarto <1.5.

Reply:
Fair enough! Thank you!